unit_tests/test_sdcm_mgmt_common.py

```diff
@@ -10,6 +10,6 @@ def test_get_manager_scylla_backend_returns_repo_address(self):  # pylint: disab
         assert url == 'http://downloads.scylladb.com/deb/ubuntu/scylla-2021.1.list'
 
     def test_get_manager_repo_from_defaults_returns_repo_address(self):  # pylint: disable=no-self-use
-        url = get_manager_repo_from_defaults("2.5", Distro.UBUNTU20)
+        url = get_manager_repo_from_defaults("3.0", Distro.UBUNTU20)
 
-        assert url == 'http://downloads.scylladb.com/deb/ubuntu/scylladb-manager-2.5-focal.list'
+        assert url == 'http://downloads.scylladb.com.s3.amazonaws.com/deb/ubuntu/scylladb-manager-3.0-focal.list'
```

Comment:
the URL without `.s3.amazonaws.com` works the same as with it

Reply:
I thought so, too. But it's actually not. Apparentely, according to israel, not using the full address can cause connection issues.
Plus, the link in the yaml literally uses the full address, so I kind of have too.